scss/form/select.scss

```diff
@@ -0,0 +1,86 @@
+.nes-select {
+  // prettier-ignore
+  $dropdown: (
+    (1,1,1,1,1,1,1),
+    (1,1,1,1,1,1,1),
+    (0,1,1,1,1,1,0),
+    (0,1,1,1,1,1,0),
+    (0,0,1,1,1,0,0),
+    (0,0,0,1,0,0,0),
+  );
+
+  position: relative;
+  width: 100%;
+
+  select {
+    width: 100%;
+    padding: 0.5rem 2.5rem 0.5rem 1rem;
+    -webkit-appearance: none;
+    appearance: none;
+    cursor: $cursor-click-url, pointer;
+    border: none;
+    border-radius: 0;
+    box-shadow: 0 4px $color-black, 0 -4px $color-black, 4px 0 $color-black, -4px 0 $color-black;
+
+    &:invalid {
+      color: $color-00;
+    }
+  }
+
+  &::after {
+    $colors: ($base-color, map-get($default-colors, "shadow"));
+
+    @include pixelize($dropdown, $colors);
+
+    position: absolute;
+    top: calc(50% - 11px);
+    right: 36px;
+    width: 2px;
+    height: 2px;
+    font-size: 2px;
+    content: "";
+    transform: scale(1.5);
+  }
+
+  &.is-success {
+    $color: map-get($success-colors, "normal");
+
+    &::after {
+      $colors: ($color, map-get($default-colors, "shadow"));
+
+      @include pixelize($dropdown, $colors);
```

Comment:
Due to how `pixelize` has been optimized to account for a default color (see #60, #143) and you are not changing the shape of the arrow icon you should not call the mixin again here. You should only need to set `color: $color` on the pseudo elements and it will just work. This will help reduce the final css size by not duplicating the long `box-shadow` declarations to create the pixel icon. As a result you will also not need to redeclare the `$colors` list above this line.

Reply:
Cool! I'll change it right away.

Reply:
No worries, thanks! I'm going to be writing up some docs/guides on how and when we should use `pixelize` so it will be more clear in the future.